spec.html

```diff
@@ -31103,7 +31103,7 @@ <h1>Properties of the _NativeError_ Prototype Objects</h1>
 
         <emu-clause id="sec-nativeerror.prototype.constructor">
           <h1>_NativeError_.prototype.constructor</h1>
-          <p>The initial value of the *"constructor"* property of the prototype for a given _NativeError_ constructor is the corresponding intrinsic object %_NativeError_% (<emu-xref href="#sec-nativeerror-constructors"></emu-xref>).</p>
+          <p>The initial value of the *"constructor"* property of the prototype for a given _NativeError_ constructor is the intrinsic object, associated with the current realm, corresponding to _NativeError_ (<emu-xref href="#table-well-known-intrinsic-objects"></emu-xref>).</p>
```

Comment:
The phrase "the current realm" isn't defined. Anyway, you don't need to bring up the realm:
```suggestion
          <p>The initial value of the *"constructor"* property of the prototype for a given _NativeError_ constructor is the constructor itself.</p>
```
E.g., the initial value of `%SyntaxError%.prototype.constructor` is `%SyntaxError%`, right?

Reply:
The term is used many times in the spec.

Reply:
You're maybe thinking of "the current Realm Record", which is used 15 times and has a definition.

But "the current realm" only appears twice, both in [6.1.7.4 Well-Known Intrinsic Objects](https://tc39.es/ecma262/#sec-well-known-intrinsic-objects), and has no definition.

(Also, "the current Realm" (without "Record") appears in the Note in [9.5.5 HostEnqueuePromiseJob](https://tc39.es/ecma262/#sec-hostenqueuepromisejob), but it's pretty clear from context that it should be "the current Realm Record".)